### PR TITLE
Renamed files, directories, and targets to match beman standard

### PR DESCRIPTION
**Pull request by camio**
[LIBRARY_NAMES]
[CMAKE.PROJECT_NAME]
[CMAKE.LIBRARY_NAME]
[CMAKE.TARGET_NAMES]
[DIRECTORY.INTERFACE_HEADERS]

### CODE REVIEW

File: CMakeLists.txt
Comment (neatudarius) [inline CMakeLists.txt:1]: LGTM :+1: 

Please check compiler explorer links included in root readme. After you merge this PR, an overnight deploy will be done automatically with compiler explorer, but we need to update our samples.

**Review by neatudarius — APPROVED** [review]

**Comment by steve-downey** [comment]
It's slightly annoying that many of the files are deleted and added instead
of being renamed, because they are short.

**Comment by camio** [comment]
I suppose to make #61 do what you want it to do, you'll have to not "squash and merge" the final result.

**Comment by steve-downey** [comment]
Since "squash merge" is another way of spelling "merge conflict", yes. It
disconnects the history more generally and prevents merging on branches.

**Comment by neatudarius** [comment]
Is this PR still applicable?

**Comment by steve-downey** [comment]
I think it's now obsolete. Landed a version that did the rename and edit in
two steps to preserve the rename in history.

**Comment by camio** [comment]
Closing as obsolete.

**Pull request closed without merging**